server/src/connectionHandlers/onCompletion.ts

```diff
@@ -268,6 +271,24 @@ function getFilePath (elementInfo: ElementInfo, fileType: string): string | unde
   return undefined
 }
 
+function convertSymbolsToCompletionItems (uri: string): CompletionItem[] {
```

Comment:
Since this is a new feature, you should point to staging-next in order not to include it in the current release cycle.

Reply:
You should also rebase on staging-next before merging